<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridMain">
     <item row="0" column="1">
      <widget class="QTabWidget" name="tabFiles">
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="edtConsole">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier 10 Pitch'; font-size:11pt; font-weight:200; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Console/&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QTreeView" name="treeFiles">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAsFile"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionCloseProject"/>
   </widget>
   <widget class="QMenu" name="menuAnalyz">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <widget class="QMenu" name="menuReport">
     <property name="title">
      <string>Reportes</string>
     </property>
     <addaction name="actionShowErrorTable"/>
     <addaction name="actionShowSymbolTable"/>
    </widget>
    <addaction name="actionBuild"/>
    <addaction name="actionRun"/>
    <addaction name="menuReport"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAnalyz"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFile">
   <property name="text">
    <string>Crear</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSaveAsFile">
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Compilar</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Ejecutar</string>
   </property>
  </action>
  <action name="actionErrores">
   <property name="text">
    <string>Errores</string>
   </property>
  </action>
  <action name="actionShowErrorTable">
   <property name="text">
    <string>Tabla de Errores</string>
   </property>
  </action>
  <action name="actionShowSymbolTable">
   <property name="text">
    <string>Tabla de Simbolos</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Cerrar Todo</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Abrir Proyecto</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Cerrar Proyecto</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
